knowledge/pop_culture/events/barbenheimer/qna.yaml

```diff
@@ -0,0 +1,61 @@
+---
+created_by: hrp27
+domain: movies
+seed_examples:
+  - answer: >
+      Both films exceeded box-office expectations. Barbie became the
+      highest-grossing domestic release in Warner Bros history, surpassing
+      Nolan's The Dark Knight. In the United States and Canada, Barbie grossed
+      $22.3 million in Thursday night previews, while Oppenheimer grossed $10.5
+      million. This was the first time two movies released in the same weekend
+      have grossed more than $10 million each in their previews. Including
+      previews, the films made $70.5 million and $33 million on their first
+      days, respectively, which led to weekend projections being increased to
+      $160 million and $77 million. The films went on to debut to $162 million
+      and $82.4 million, respectively.
+    question: what is the Box office revenue of both the  Films?
```

Comment:
This question is too broad, you need to specifically say Barbie and Oppenheimer in this question, otherwise the model doesn't know what films you are talking about.